### DISCUSSION TITLE
Anything special for 1000 subscribers?

### DISCUSSION THREAD

**Discussion by niosus** [cpp-for-yourself/lectures-and-homeworks#39] [Polls]
I am amazed that there are already nearly 1000 people subscribed to my channel. As we are now at 991, I assume soon we will reach the 1000 mark 🤯 

So, any ideas on anything special I should do? Please answer the poll here and comment below with any suggestions.

**Comment by Ermanas** [comment]
Now it's more than 2500 people :partying_face:

**Reply by niosus** [reply]
Yeah, I'm pretty impressed myself 🙌

**Reply by Ermanas** [reply]
The way you teach is pretty impressive actually. In the class, you put a lot of effort to let the students understand. You always try to understand what is clear and what is not. I really respect your this attitude. Also when I see the system that you created for homeworks, your passion on system design is pretty obvious, great work! 

And the Q&A was great at 1000 subs. I'm pretty curious what'll be next for the 10k subs :)

**Reply by niosus** [reply]
Thanks for the kind words! I'm happy that my efforts are appreciated! 🙏
